### PR TITLE
muktamahee: v2.538 added

### PR DESCRIPTION
**Pull request by m4rc1e**
@girish-dalvi These fonts are a significant improvement. Thank you for your hard work.

The DESCRIPTION.en_us.html for all the Mukta projects may need an update.

### CODE REVIEW

File: ofl/muktamahee/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<p>
+Mukta is a Unicode compliant, versatile, contemporary, humanist, mono-linear typeface available in seven weights, supporting Devanagari, Gujarati, Tamil and Latin scripts. 
+This type family is a libre licensed version of Ek's self-titled multi-script project, an ongoing effort to develop a unified type family for each Indian script. 
+The goal is to build one harmonious family across all Indian scripts without letting the visual features of one script dominate over others. 
+This ensures that the fonts can be used successfully for both single and multi-script purposes. 
+</p>
+<p>
```
Comment (girish-dalvi) [inline ofl/muktamahee/DESCRIPTION.en_us.html]: Mukta Mahee was designed by Shuchita Grover and Noopur Datye.

File: ofl/muktamahee/DESCRIPTION.en_us.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Mukta was designed by Girish Dalvi and Yashodeep Gholap.
```
Comment (girish-dalvi) [inline ofl/muktamahee/DESCRIPTION.en_us.html]: <a href="https://fonts.google.com/specimen/Mukta"> Mukta </a> was designed by Girish Dalvi and Yashodeep Gholap.

File: ofl/muktamahee/DESCRIPTION.en_us.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<p>
+Mukta is a Unicode compliant, versatile, contemporary, humanist, mono-linear typeface available in seven weights, supporting Devanagari, Gujarati, Tamil and Latin scripts.
```
Comment (girish-dalvi) [inline ofl/muktamahee/DESCRIPTION.en_us.html]: Mukta is a Unicode compliant, versatile, contemporary, humanist, mono-linear typeface family available in seven weights, supporting Devanagari, Gujarati, Gurumukhi, Tamil and Latin scripts.

**Comment by davelab6** [comment]
> Minor changes requested to the description file. I've made changes and posted them as comments. 

Done :)

**Pull request merged**